src/gentropy/common/spark_helpers.py

```diff
@@ -543,3 +542,28 @@ def safe_array_union(a: Column, b: Column) -> Column:
     return f.when(a.isNotNull() & b.isNotNull(), f.array_union(a, b)).otherwise(
         f.coalesce(a, b)
     )
+
+
+def create_empty_column_if_not_exists(
+    col_name: str, col_schema: t.DataType = t.NullType()
```

Comment:
I have a bad feeling about `col_schema: t.DataType = t.NullType()`. I think if you just leave it default, you won't be able to write:

```python
(
    spark.createDataFrame([('a',),('b',)], ['col1'])
    .select(create_empty_column_if_not_exists('testCol'))
    .show()
)
```

So, this looks good:

```
+-------+
|testCol|
+-------+
|   NULL|
|   NULL|
+-------+
```

However:
```python
(
    spark.createDataFrame([('a',),('b',)], ['col1'])
    .select(create_empty_column_if_not_exists('testCol'))
    .write.parquet('gs://ot-team/dsuveges/test.parquet')
)
```
gives:
```
AnalysisException: [UNSUPPORTED_DATA_TYPE_FOR_DATASOURCE] The Parquet datasource doesn't support the column `testCol` of the type "VOID".
```

Reply:
Null columns are coalesced to create a valid VCF, so technically we won't be writing any null columns

Reply:
Very nice comment though, I didn't know that